### ISSUE TITLE
Custom filters causing failures for rollbar-test rake task

### ISSUE THREAD

**Issue by patricklindsay** [rollbar/rollbar-gem#499]
I have a before_filter on my `ApplicationController` which is causing the rollbar-test to fail as it redirects the request.

Why does the `RollbarTestController` need to actually inherit from `ApplicationController`? If it was inheriting from `ActionController::Base` custom filters would not effect the test.

Two possible solutions;
1. Set `RollbarTestController` to inherit from `ActionController::Base`
2. Set `RollbarTestController` to inherit from a configurable controller and set default to `ApplicationController`

[RollbarTestController](https://github.com/rollbar/rollbar-gem/blob/master/lib/rollbar/rake_tasks.rb#L55)
```
Testing manual report...
[Rollbar] Scheduling item
[Rollbar] Sending item
[Rollbar] Success
[Rollbar] Details: https://rollbar.com/instance/uuid?uuid=x-y (only available if report was successful)
Setting up the controller.
Processing...
Started GET "/verify" for  at 2016-07-21 07:24:05 +0000
Processing by RollbarTestController#verify as HTML
Redirected to https://www.example.com/en/verify
Filter chain halted as :locale_redirect rendered or redirected
Completed 301 Moved Permanently in 10ms (ActiveRecord: 0.0ms)
Test failed! You may have a configuration issue, or you could be using a gem that's blocking the test. Contact support@rollbar.com if you need help troubleshooting.
```

**Comment by jondeandres** [comment]
Hey @patricklindsay, thanks for reporting this. We'll probably change `RollbarTestController` to inherit from `ActionController::Base`.

Thanks!

**Comment by florish** [comment]
Just to inform others: I'm running into a similar issue just now, with the cause being a `before_action` doing an authentication check, in my case using the `devise` gem. The 401 response generated by this also gives the "Test failed!" message when running `rake rollbar:test`.

**Comment by jondeandres** [comment]
Hi, I've just pushed a commit making the test controller we build to inherit from `ActionController::Base`. I think your problems should be fixed. I've released v2.13.1 with this fix and few minor fixes we pushed last week.

Thanks both.

**Comment by florish** [comment]
@jondeandres Thanks for fixing this! I can confirm this fixes the issue I had, both errors generated by `rake rollbar:test` now get reported.
